app/components/searchworks4/theses_dissertation_facet_header_component.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="p-2 mb-2 d-flex justify-content-center" data-controller='facet-checkboxes'>
+  <button class="btn btn-outline-secondary p-3 position-relative">
+    <%= label_tag("facet_option_stanford_only") do %>
+      <%= check_box(
+            'f',
+            'stanford_only',
+            {
+              multiple: true,
+              id: "facet_option_stanford_only",
+              checked: stanford_only_selected?,
+              data: { action: 'facet-checkboxes#toggleCheckbox' }
+            },
+            "Show Stanford work only",
+            nil
+          )
+      %>
+      <%= link_to(
+            stanford_only_selected? ? all_theses_dissertations_url : stanford_only_url ,
```

Comment:
I think this ought to be using FacetItemPresenter#href. https://github.com/projectblacklight/blacklight/blob/508f67a95ea1a53884f199ef594b8aa6e8fb9483/app/presenters/blacklight/facet_item_presenter.rb#L81-L87